### PR TITLE
[SIG-4116] Replace incident overview table header

### PR DESCRIPTION
**Pull request by janjaap**
This PR replaces the table header sorting options with the Sort component

### CODE REVIEW

**Comment by kiruys** [comment]
The date ordering seems to be reversed, so 'nieuw - oud' gives old incidents on top, and 'oud - nieuw' gives new incidents on top

**Comment by kiruys** [comment]
I'm not sure what "toegewezen aan" ordering does exactly.

File: src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.tsx [hunk 1]
```diff
@@ -137,55 +109,23 @@ const List: FunctionComponent<ListProps> = ({
...
-            <ThSubcategory
-              data-testid="sortSubcategory"
-              onClick={onSort('sub_category,-created_at')}
-            >
-              Subcategorie {renderChevron('sub_category')}
+            <Th data-testid="sortId">Id</Th>
+            <Th data-testid="sortDaysOpen">Dag</Th>
+            <ThDate data-testid="sortCreatedAt">Datum en tijd</ThDate>
+            <ThSubcategory data-testid="sortSubcategory">
+              Subcategorie
```
Comment (kiruys) [inline src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.tsx:116]: Is it possible to remove the data-testid attributes now that the sorting is done differently?

**Review by kiruys — APPROVED** [review]

**Comment by janjaap** [comment]
> I'm not sure what "toegewezen aan" ordering does exactly.

Good point; I forgot to make that option conditional base on the value of the `assignSignalToEmployee` feature flag.

**Comment by janjaap** [comment]
> The date ordering seems to be reversed, so 'nieuw - oud' gives old incidents on top, and 'oud - nieuw' gives new incidents on top

Ouch, good one; I'll change that.

File: src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.tsx (hunk 1 above)
Comment (janjaap) [reply src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.tsx:116]: I guess so. They don't seem to be used at all.

**Pull request merged**